reledmac.dtx

```diff
@@ -13913,7 +13913,7 @@
 % \end{macro}
 % \end{macro}
 %
-% \subsection{Patching standard output's commands}
+% \subsection{Patching output's standard commands}
```

Comment:
no. It is the command for the standard output, not the standard commands for the output